src/index.js

```diff
@@ -1,11 +1,23 @@
 import './style.css';
+// import arraydef from './arraydef';
 import arrayDefault from './default';
+import getLocalStorage from './refresh';
 
-const getTheArra = arrayDefault();
-let count = 0;
-getTheArra.forEach((Number) => {
+const getArray = [];
+document.getElementById('add_game').addEventListener('submit', (event) => {
+  event.preventDefault();
+  const getElements = document.getElementById('recent').childElementCount;
+  const getName = event.target[0].value;
+  const getScore = event.target[1].value;
+  arrayDefault(getName, getScore);
+  const game = {
+    name: getName,
+    score: getScore,
+  };
+  getArray.push(game);
+  let count = getElements;
```

Comment:
- [ ] I noticed that when I click the submit button, it immediately displays the score in the DOM which is not the desired result. The submit button should only add the record to the API (Hit the API) and not display scores. KIndly fix that 👍🏽 

![ezgif com-gif-maker (1)](https://user-images.githubusercontent.com/63473584/186513184-eb44dd71-1c65-4571-8ccb-9344ac0d975c.gif)

Reply:
Your right now it works like expected